MISC/SetWeatherTypeNowPersist.md

```diff
@@ -8,7 +8,7 @@ ns: MISC
 void SET_WEATHER_TYPE_NOW_PERSIST(char* weatherType);
 ```
 
-Refer to [`SET_WEATHER_TYPE_NOW`](#0x29B487C359E19889) for weather types.
+Refer to [`SET_WEATHER_TYPE_NOW`](#_0x29B487C359E19889) for weather types.
```

Comment:
The ones missing the underscore (`_`) were addressed in the following [PR](https://github.com/citizenfx/natives/pull/918/files).